go-controller/pkg/ovn/egressip_test.go

```diff
@@ -3410,6 +3410,167 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations cluster default network"
+				egressIP := net.ParseIP("192.168.126.211")
+				_, nodeSubnetV4, _ := net.ParseCIDR(v4Node1Subnet)
+				_, nodeSubnetV6, _ := net.ParseCIDR(v6Node1Subnet)
+
+				annotations := map[string]string{
+					"k8s.ovn.org/node-primary-ifaddr":             fmt.Sprintf("{\"ipv4\": \"%s\", \"ipv6\": \"%s\"}", nodeIPv4, ""),
+					"k8s.ovn.org/node-subnets":                    fmt.Sprintf("{\"default\":\"%s\",\"%s\"}", v4Node1Subnet, v6Node1Subnet),
+					"k8s.ovn.org/node-transit-switch-port-ifaddr": "{\"ipv4\":\"100.88.0.2/16\", \"ipv6\": \"fd97::2/64\"}",
+					util.OVNNodeHostCIDRs:                         fmt.Sprintf("[\"%s\"]", nodeIPv4),
+					"k8s.ovn.org/zone-name":                       node1Name,
+				}
+				node := getNodeObj(node1Name, annotations, map[string]string{}) // add node to avoid errori-ing out on transit switch IP fetch
+				fakeOvn.startWithDBSetup(
+					libovsdbtest.TestSetup{
+						NBData: []libovsdbtest.TestData{
+							&nbdb.LogicalRouterPort{
+								UUID:     types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node1Name + "-UUID",
+								Name:     types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node1Name,
+								Networks: []string{nodeLogicalRouterIfAddrV6, nodeLogicalRouterIfAddrV4},
+							},
+							&nbdb.LogicalRouter{
+								Name: types.OVNClusterRouter,
+								UUID: types.OVNClusterRouter + "-UUID",
+							},
+							&nbdb.LogicalRouter{
+								Name:    types.GWRouterPrefix + node1Name,
+								UUID:    types.GWRouterPrefix + node1Name + "-UUID",
+								Ports:   []string{types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node1Name + "-UUID"},
+								Options: map[string]string{"dynamic_neigh_routers": "false"},
+							},
+							&nbdb.LogicalSwitchPort{
+								UUID: "k8s-" + node.Name + "-UUID",
+								Name: "k8s-" + node.Name,
+								Addresses: []string{"fe:1a:b2:3f:0e:fb " + util.GetNodeManagementIfAddr(nodeSubnetV4).IP.String(),
+									"fe:1a:b2:3f:0e:fb " + util.GetNodeManagementIfAddr(nodeSubnetV6).IP.String()},
+							},
+							&nbdb.LogicalSwitch{
+								UUID:  node.Name + "-UUID",
+								Name:  node.Name,
+								Ports: []string{"k8s-" + node.Name + "-UUID"},
+							},
+						},
+					},
+					&corev1.NamespaceList{
+						Items: []corev1.Namespace{*egressNamespace},
+					},
+					&corev1.PodList{
+						Items: []corev1.Pod{egressPod},
+					},
+					&corev1.NodeList{
+						Items: []corev1.Node{node},
+					},
+				)
+				// matches only on namespace
+				eIP := egressipv1.EgressIP{
+					ObjectMeta: newEgressIPMeta(egressIPName),
+					Spec: egressipv1.EgressIPSpec{
+						EgressIPs: []string{
+							egressIP.String(),
+						},
+						NamespaceSelector: metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"name": egressNamespace.Name,
+							},
+						},
+						PodSelector: metav1.LabelSelector{
+							MatchLabels: egressPodLabel,
+						},
+					},
+				}
+				i, n, _ := net.ParseCIDR(podV4IP + "/23")
+				n.IP = i
```

Comment:
imho, these could be better named. Or even better wrapped up in an helper func somewhere. 

Actually, I think there's already an helper out there for this.